extensions/banner/runtime/src/main/java/io/quarkus/banner/runtime/BannerConfig.java

```diff
@@ -0,0 +1,14 @@
+package io.quarkus.banner.runtime;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "banner", phase = ConfigPhase.BUILD_TIME)
```

Comment:
```suggestion
@ConfigRoot(name = "banner")
```

nitpick: the default phase is `BUILD_TIME`

Reply:
I personally like having the phase explicitly set, but yeah it's just a matter of preference :)

Reply:
Yes, it is a minor one. We can leave it the way it is :-)